buckaroo/Bash.fs

```diff
@@ -54,16 +54,16 @@ let runBashSync (command : String) (stdoutHandler : ProgressCallback) (stderrHan
     p.CancelOutputRead()
     p.CancelErrorRead()
 
-  let! exitSignal = 
+  let! exitSignal =
     Task.Factory.StartNew(startProcess)
     |> Async.AwaitTask
     |> Async.StartChild
 
   do! exitSignal
 
-  if p.ExitCode > 0
-  then 
-    return 
+  if p.ExitCode > 0 && p.ExitCode < 128
```

Comment:
I think we should leave it up to the caller if they want to throw or not. Does 128 always indicate success? I think this depends on the command you have run.

Reply:
this is a workaround for a bug which will be addressed in a seperate PR